nonce/nonce.go

```diff
@@ -194,24 +209,24 @@ func (ns *NonceService) Nonce() (string, error) {
 func (ns *NonceService) Valid(nonce string) bool {
 	c, err := ns.decrypt(nonce)
 	if err != nil {
-		ns.stats.Inc("Invalid.Decrypt", 1)
+		ns.nonceValidations.WithLabelValues("invalid, decrypt").Inc()
```

Comment:
I think this is another case where there should be two labels, one for the result and one for the error type/class.